scripts/env-data.sh

```diff
@@ -6,10 +6,13 @@ DEFAULT_DATADIR="/var/lib/postgresql/${POSTGRES_MAJOR_VERSION}/main"
 # Commented for documentation. You can specify the location of
 # pg_wal directory/volume using the following environment variable:
 # POSTGRES_INITDB_WALDIR (default value is unset)
+DEFAULT_SCRIPTS_LOCKFILE_DIR='/docker-entrypoint.initdb.d'
```

Comment:
Perhaps use double quotes here, so it's the same as the others

Reply:
fixed